gui/webdisplay/src/TWebWindowsManager.cxx

```diff
@@ -431,6 +449,15 @@ bool ROOT::Experimental::TWebWindowsManager::Show(ROOT::Experimental::TWebWindow
 
       TestProg(prog, gEnv->GetValue("WebGui.Firefox", ""));
 
+#ifdef _MSC_VER
+      TString ProgramFiles = gSystem->Getenv("ProgramFiles");
+      ProgramFiles.ReplaceAll(" (x86)", "");
+      TString ProgramFilesx86 = gSystem->Getenv("ProgramFiles(x86)");
+      if (ProgramFilesx86.Length() && !gSystem->AccessPathName(Form("%s\\Mozilla Firefox\\firefox.exe", ProgramFilesx86.Data())))
+         prog = TString::Format("%s\\Mozilla Firefox\\firefox.exe", ProgramFilesx86.Data());
```

Comment:
Please use here call like:
if (ProgramFilesx86.Length())
    TestProg(prog, TString::Format("%s\\Mozilla Firefox\\firefox.exe", ProgramFilesx86.Data()));

Inside TestProg I check if executable is existing.
And if program already found (or configured by user) no new tests are required

Reply:
OK, I will do it that way